Homework/text_box_page.py

```diff
@@ -0,0 +1,28 @@
+from selenium.webdriver.common.by import By
+
+
+class TextBoxPage:
+    def __init__(self, driver):
+        self.driver = driver
+
+    def enter_full_name(self, full_name):
```

Comment:
The `enter_full_name`, `enter_email`, `enter_current_address`, and `enter_permanent_address` could be extended to handle input validation and normalization (e.g., trim leading and trailing whitespace). This is a good practice.